b2/bucket.py

```diff
@@ -368,8 +368,14 @@ def _upload_large_file(
         # Select the part boundaries
         part_ranges = choose_part_ranges(content_length, minimum_part_size)
 
-        # Tell B2 we're going to upload a file
-        unfinished_file = self.start_large_file(file_name, content_type, file_info)
+        # Check for unfinished files with same name
+        (unfinished_file, finished_parts) = self._find_unfinished_file(
```

Comment:
unnecessary parenthesis (they make yapf slower and the code less readable (for me))